### PR TITLE
[SERV-737] Add "hello world" VueJS app

### PR DESCRIPTION
**Pull request by markmatney**
The app was scaffolded according to the instructions [here](https://vuejs.org/guide/quick-start.html#try-vue-online). To build locally, you must have Node 16.0 or higher installed.

To view the front-end in a browser, run the harvester app locally:

```bash
# shutdown later with: `mvn docker:stop`
mvn clean pre-integration-test -Dmaven.surefire.skip
```

then navigate to `localhost:${test.harvester.port}/admin`. Note the sweet favicon ✨

### CODE REVIEW

File: src/main/frontend/.vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["Vue.volar", "Vue.vscode-typescript-vue-plugin"]
```
Comment (ksclarke) [inline src/main/frontend/.vscode/extensions.json:2]: Why the exclusion for this one .vscode file?

**Review by ksclarke — APPROVED** [review]

File: src/main/frontend/.prettierrc.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (DRickard) [inline src/main/frontend/.prettierrc.json:1]: FYI, GitHub wants a EOF newline

**Review by DRickard — APPROVED** [review]

File: src/main/frontend/.vscode/extensions.json (hunk 1 above)
Comment (markmatney) [reply src/main/frontend/.vscode/extensions.json:2]: No idea. Everything in src/main/frontend was autogenerated by `npm init vue@latest`, and all except for public/favicon.ico and package.json were left unmodified by me.

File: src/main/frontend/.prettierrc.json (hunk 2 above)
Comment (markmatney) [reply src/main/frontend/.prettierrc.json:1]: This was an auto-generated file and I didn't touch it, but if I do I'll make sure it looks good.

**Pull request merged**